mqtt-streaming/src/test/java/docs/javadsl/MqttFlowTest.java

```diff
@@ -149,7 +149,7 @@ public void establishServerBidirectionalConnectionAndSubscribeToATopic()
                       run =
                           Source.<Command<Object>>queue(2, OverflowStrategy.dropHead())
                               .via(mqttFlow)
-                              .toMat(BroadcastHub.of(DecodeErrorOrEvent.class), Keep.both())
+                              .toMat(BroadcastHub.of(DecodeErrorOrEvent.classOf()), Keep.both())
```

Comment:
JFYI `BroadcastHub.of` doesn't even use the class value.  It's only there to help define the type parameter.  So you can actually pass `null` and let type inference define type parameter.  😄